src/components/Header/index.js

```diff
@@ -125,6 +136,80 @@ const HeaderLink = ({ to, onClick, children, location, activePath = [] }) => {
   );
 };
 
+let HeaderDropDownLink = ({
+  to = [],
+  onClick,
+  children,
+  location,
+  activePath = [],
```

Comment:
`activePath` is needed?

Reply:
yeah i just forgot to implement it :) good catch!